spinq10q.py

```diff
@@ -63,48 +72,61 @@ def create(runcard_path=RUNCARD):
     instruments.update(modules)
     instruments = load_instrument_settings(runcard, instruments)
 
+    # DEBUG: debug folder = report folder
+    import os
+    from datetime import datetime
+
+    QPU = "spinq10q"
+    debug_folder = f"/home/users/alvaro.orgaz/reports/{datetime.now().strftime('%Y%m%d')}_{QPU}_/debug/"
+    if not os.path.exists(debug_folder):
+        os.makedirs(debug_folder)
+    for name in modules:
+        modules[name]._debug_folder = debug_folder
+
     # Create channel objects
-    channels = ChannelMap()
-    # Readout
-    channels |= Channel(name="L3-20", port=modules["qrm_rf0"].ports["o1"])
-    channels |= Channel(name="L3-21", port=modules["qrm_rf1"].ports["o1"])
-    # Feedback
-    channels |= Channel(name="L1-1", port=modules["qrm_rf0"].ports["i1"])
-    channels |= Channel(name="L2-17", port=modules["qrm_rf1"].ports["i1"])
-    # Drive
-    channels |= Channel(name="L6-1", port=modules["qcm_rf0"].ports["o1"])
-    channels |= Channel(name="L6-2", port=modules["qcm_rf0"].ports["o2"])
-    channels |= Channel(name="L6-3", port=modules["qcm_rf1"].ports["o1"])
-    channels |= Channel(name="L6-4", port=modules["qcm_rf1"].ports["o2"])
-    channels |= Channel(name="L6-5", port=modules["qcm_rf2"].ports["o1"])
-    channels |= Channel(name="L6-6", port=modules["qcm_rf2"].ports["o2"])
-    channels |= Channel(name="L6-7", port=modules["qcm_rf3"].ports["o1"])
-    channels |= Channel(name="L6-8", port=modules["qcm_rf3"].ports["o2"])
-    channels |= Channel(name="L6-9", port=modules["qcm_rf4"].ports["o1"])
-    channels |= Channel(name="L6-10", port=modules["qcm_rf4"].ports["o2"])
-    # Flux
-    channels |= Channel(name="L6-39", port=modules["qcm_bb0"].ports["o1"])
-    channels |= Channel(name="L6-40", port=modules["qcm_bb0"].ports["o2"])
-    channels |= Channel(name="L6-41", port=modules["qcm_bb0"].ports["o3"])
-    channels |= Channel(name="L6-42", port=modules["qcm_bb0"].ports["o4"])
-    channels |= Channel(name="L6-43", port=modules["qcm_bb1"].ports["o1"])
-    channels |= Channel(name="L6-44", port=modules["qcm_bb1"].ports["o2"])
-    channels |= Channel(name="L6-45", port=modules["qcm_bb1"].ports["o3"])
-    channels |= Channel(name="L6-46", port=modules["qcm_bb1"].ports["o4"])
-    # Qubit 9-10 are not connected to flux
-    channels |= Channel(name="L6-47", port=None)
-    channels |= Channel(name="L6-48", port=None)
+    channels = {}
```

Comment:
Why using the dictionariesinstead of `ChannelMap()` object that prevent from typo since there is no need to specify twice the channel name?